Broken AppVeyor build

Currenly the appveyor build is failing ...

See for instance https://ci.appveyor.com/project/serilog-web/classic-webapi/branch/master 

```
Classic\WebApi\Enrichers\BaseWebApiContextInfoEnricher.cs(22,51): error CS1525: Invalid expression term 'throw' [C:\projects\classic-webapi\src\SerilogWeb.Classic.WebApi\SerilogWeb.Classic.WebApi.csproj]
Classic\WebApi\Enrichers\BaseWebApiContextInfoEnricher.cs(22,51): error CS1002: ; expected [C:\projects\classic-webapi\src\SerilogWeb.Classic.WebApi\SerilogWeb.Classic.WebApi.csproj]
Attempting to build package from 'SerilogWeb.Classic.WebApi.csproj'.
MSBuild auto-detection: using msbuild version '14.0' from 'C:\Program Files (x86)\MSBuild\14.0\bin\amd64'.
The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Unable to find 'C:\projects\classic-webapi\src\SerilogWeb.Classic.WebApi\bin\Release\SerilogWeb.Classic.WebApi.dll'. Make sure the project has been built.
```

This looks like a problem with using recent C# features in : 

https://github.com/serilog-web/classic-webapi/blob/d7042f54f7685ca4ada23c679ae8596cb116785b/src/SerilogWeb.Classic.WebApi/Classic/WebApi/Enrichers/BaseWebApiContextInfoEnricher.cs#L22

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,0 +1,12 @@
+version: '{build}'
+build_script:
+- ps: ./Build.ps1 -majorMinor "2.0" -patch "$env:APPVEYOR_BUILD_VERSION" -customLogger "C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+artifacts:
+- path: SerilogWeb.*.nupkg
+deploy:
+- provider: NuGet
+  api_key:
+    secure: RKWJyeIQtpcfdYoghDQ46d1BnP9ouE5bifxbxVLpV1jW/hfu9BartYv9RYi5kZch
+  skip_symbols: true
+  on:
+    branch: master

--- a/serilog-web-classic-webapi.sln
+++ b/serilog-web-classic-webapi.sln
@@ -1,11 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.12
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{06F15C5E-4564-4257-9E40-C1FBD5A26CB8}"
 	ProjectSection(SolutionItems) = preProject
+		appveyor.yml = appveyor.yml
 		Build.ps1 = Build.ps1
+		CHANGES.md = CHANGES.md
 		README.md = README.md
 	EndProjectSection
 EndProject
@@ -38,5 +40,8 @@ Global
 	GlobalSection(NestedProjects) = preSolution
 		{A0C38DC7-1D9F-4D0C-8EFA-CEEE34FCA188} = {AF1B47E8-4D83-4DD6-8D10-69A791F886F0}
 		{F6EA68EE-432C-414D-8451-67B274FEA5F3} = {20BE9067-F8FD-47C3-9016-43A4690FA825}
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {CE15C6CE-BD16-4386-B728-F00B26AC6005}
 	EndGlobalSection
 EndGlobal

